notebooks/community/matching_engine/sdk_matching_engine_for_indexing.ipynb

```diff
@@ -98,10 +98,10 @@
         "## Before you begin\n",
```

Comment:
<p>I'd change this to "Rerun the $ gcloud auth login in&nbsp;the Vertex AI Workbench notebook terminal when you are logged out and need the credential again.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>